apps/docs/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@rnr/docs",
+  "type": "module",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro check && astro build",
+    "preview": "astro preview",
+    "astro": "astro"
```

Comment:
To be able to clean the document side 

`"clean": "rm -rf .turbo node_modules dist .astro"
`
might be useful to add.